packages/grafana-e2e/src/flows/addDashboard.ts

```diff
@@ -1,21 +1,17 @@
 import { e2e } from '../index';
 import { Url } from '../support/url';
 
-export const addDashboard = async (): Promise<{ dashboardTitle: string; uid: string }> => {
+export const addDashboard = async () => {
   e2e().logToConsole('Adding dashboard');
   e2e.pages.AddDashboard.visit();
 
   const dashboardTitle = e2e.flows.saveNewDashboard();
   e2e().logToConsole('Added dashboard with title:', dashboardTitle);
 
-  return new Promise(resolve => {
-    e2e()
-      .url()
-      .then((url: string) => {
-        resolve({
-          dashboardTitle,
-          uid: Url.getDashboardUid(url),
-        });
-      });
-  });
+  e2e()
```

Comment:
Wouldn't it be possible to use `await` here instead of chaining a promise?

Reply:
Probably, but it seems like Cypress wants to handle promises their way, I got into other issues when using `async` here and in beforeAll. So removing all async seems to solve this. See https://github.com/cypress-io/cypress/issues/1417

Reply:
Aha, so it is not a promise. I understand. Pretty bad naming of the cypress team.